bindings/python/tests/test_generate.py

```diff
@@ -0,0 +1,116 @@
+from context import scripts
+import scripts.generate as generate
+import test_parse
+
+
+"""
+TODO
```

Comment:
Please use sphinx supported semantics:

https://www.sphinx-doc.org/en/master/usage/extensions/todo.html

Also, add all TODOs as issues on PCL too (1 per area of interest)